js-project-guidelines.md

```diff
@@ -67,6 +69,16 @@ When reporting a bug, if possible, provide a way for us to reproduce it (or even
 
 Always run tests before pushing and PR'ing your code.
 
+### Captains and Maintainers
+
+For each JavaScript repository in the IPFS, libp2p, IPLD, or related GitHub orgs, there should be a single Captain specified in the Maintainers section of the README for that repository. The Captain is in charge of merging PRs, keeping issues up to date, and overall quality of a repository.
+
+Sometimes, a Captain may elect to have other maintainers that also have merge ability and commit access to the main repo. These maintainers can help out, but defer to the Captain as the person in charge of maintaining quality across the repo. It is important that this distinction is explicit; if there are long-standing PRs or issues, it is ultimately up to the Captain to gather information about the issue or PR, and to merge them. Our community strives to trust the Captain as someone who ultimately has the most knowledge of a repo (even if they are also opinionated). This may change in the future, if we go with more non-hierarchical model.
```

Comment:
It would be good to explicitly mention that a Captain only makes a decision if he needs to and all methods of discussion are exhausted.